editor/src/core/workers/parser-printer/parser-printer-exports.spec.tsx

```diff
@@ -286,28 +286,7 @@ export function App() {
         ]
       `)
     } else {
-      fail('Did not parse successfully.')
-    }
-  })
-
-  it(`parses 'export function' marked values`, () => {
```

Comment:
Why was this test removed?

Reply:
it seemed like not only the name was redundant, but it was the same test as the `it` before it

Reply:
Oh wow! It certainly looks like it was copy pasta'd